draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -262,17 +285,16 @@
       meter, <xref target="RFC2698">Two Rate Tri Color Marking
       (trTCM)</xref> meter, and Single Rate Two Color Marking meter.
       Different vendors can extend it with other types of meters as
-      well.
-      </t>
+      well.</t>
```

Comment:
Add a statement here that says, that Meters are defined in the QoS Actions module.